tests/svntests.py

```diff
@@ -18,9 +18,9 @@ class SvnTests(GitSvnTests):
     """
 
     scm = 'svn'
-    initial_clone_command = 'svn (co|checkout) '
-    update_cache_command  = 'svn up(date)?'
-    sslverify_false_args  = '--trust-server-cert'
+    initial_clone_command = 'svn.*(co|checkout) '
+    update_cache_command = 'svn.*up(date)?'
+    sslverify_false_args = '--trust-server-cert'
```

Comment:
I applied your suggestions and created a new pull request (#121).
Please disconsider this one.